tmt/utils.py

```diff
@@ -6183,3 +6190,34 @@ def __exit__(self, *args: Any) -> None:
     @property
     def duration(self) -> datetime.timedelta:
         return self.end_time - self.start_time
+
+
+def retry(
+        func: Callable[..., T],
+        retries: int,
+        delay: int,
```

Comment:
In the `podman` plugin we use `interval` and `attempts`. What about using those here as well for consistency?